src/main/resources/mixins.reach-entity-attributes.json

```diff
@@ -1,6 +1,6 @@
 {
   "minVersion": "0.8",
-  "compatibilityLevel": "JAVA_8",
+  "compatibilityLevel": "JAVA_14",
```

Comment:
Target compatibility level should match the requirement of the game and launcher.

Reply:
Not possible I am afraid, the `MixinEnvironment.CompatibilityLevel` enum only goes up to JAVA_14.

Reply:
Nevermind, it was updated since last time I checked.